en/console-and-shells/completion-shell.rst

```diff
@@ -54,14 +54,47 @@ For example::
 
 Returns::
 
-    --help -h --verbose -v --quiet -q --connection -c --template -t
+    --help -h --verbose -v --quiet -q --everything --connection -c --force -f --plugin -p --prefix --theme -t
+Vous pouvez passer un autre argument représentant une sous-commande du shell :
+cela vous retournera les options spécifiques à cette sous-commande.
 
-Bash Example
-============
+You can also pass an additional argument being the shell sub-command : it will
+output the specific options of this sub-command.
+
+How to enable Bash autocompletion for the CakePHP Console
+=========================================================
+
+Using a Debian distribution
+---------------------------
+
+First, make sure the **bash-completion** library is installed.
+Create a file named **cake** in **/etc/bash_completion.d/** and put the
+:ref:`bash-completion-file-content` inside it.
+
+Save the file, then restart your console.
+
+Using Mac OS X
+--------------
+
+First, make sure the **bash-completion** library is installed.
+You can install it using **homebrew** with the command
+``brew install bash-completion``.
+Create a file named **cake** in **/usr/local/etc/bash_completion.d/** and put
+the :ref:`bash-completion-file-content` inside it.
 
-The following bash example comes from the original author::
+Save the file, then restart your console.
```

Comment:
@HavokInspiration Just thinking out load but as those two paragraph are almost exactly the same, maybe it will be possible to squash them, add a link the the https://bash-completion.alioth.debian.org/ and only add a ``.. note::`` at the end that for Mac OS X, this librairie is available via homebrew by typing ``brew install bash-completion`` or something similar ?